src/Raven.Server/Files/FileSystemFile.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using Sparrow.Json;
+using Voron.Data;
+
+namespace Raven.Server.Files
+{
+    public class FileSystemFile
```

Comment:
That won't be a problem for us. If we have any usage of those two in the same class (I don't think so) we can use full qualified names then.